Task :compileJava FAILED

Don't have any experience with Java, do I need a specific version to compile this extension?

Java version:
```
% java -version
openjdk version "23.0.2" 2025-01-21
OpenJDK Runtime Environment Homebrew (build 23.0.2)
OpenJDK 64-Bit Server VM Homebrew (build 23.0.2, mixed mode, sharing)
```

Gradle version:
```
% gradle -version

------------------------------------------------------------
Gradle 8.13
------------------------------------------------------------

Build time:    2025-02-25 09:22:14 UTC
Revision:      073314332697ba45c16c0a0ce1891fa6794179ff

Kotlin:        2.0.21
Groovy:        3.0.22
Ant:           Apache Ant(TM) version 1.10.15 compiled on August 25 2024
Launcher JVM:  23.0.2 (Homebrew 23.0.2)
Daemon JVM:    /opt/homebrew/Cellar/openjdk/23.0.2/libexec/openjdk.jdk/Contents/Home (no JDK specified, using current Java home)
OS:            Mac OS X 14.4 aarch64
```

Error log:
```
% gradle -PGHIDRA_INSTALL_DIR=/opt/homebrew/Caskroom/ghidra/11.3.1-20250219/ghidra_11.3.1_PUBLIC

> Task :compileJava
/Users/gavin/Git/QtREAnalyzer/QtREAnalzyer/src/main/java/qtreanalzyer/BytesExprPcodeEmulator.java:69: error: ExprPcodeArithmetic is not abstract and does not override abstract method modAfterLoad(int,AddressSpace,Expr,int,Expr) in PcodeArithmetic
enum ExprPcodeArithmetic implements PcodeArithmetic<Expr> {
^
/Users/gavin/Git/QtREAnalyzer/QtREAnalzyer/src/main/java/qtreanalzyer/BytesExprPcodeEmulator.java:124: error: method does not override or implement a method from a supertype
        @Override
        ^
/Users/gavin/Git/QtREAnalyzer/QtREAnalzyer/src/main/java/qtreanalzyer/BytesExprPcodeEmulator.java:130: error: method does not override or implement a method from a supertype
        @Override
        ^
3 errors

> Task :compileJava FAILED

[Incubating] Problems report is available at: file:///Users/gavin/Git/QtREAnalyzer/QtREAnalzyer/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Compilation failed; see the compiler output below.
  /Users/gavin/Git/QtREAnalyzer/QtREAnalzyer/src/main/java/qtreanalzyer/BytesExprPcodeEmulator.java:69: error: ExprPcodeArithmetic is not abstract and does not override abstract method modAfterLoad(int,AddressSpace,Expr,int,Expr) in PcodeArithmetic
  enum ExprPcodeArithmetic implements PcodeArithmetic<Expr> {
  ^
  /Users/gavin/Git/QtREAnalyzer/QtREAnalzyer/src/main/java/qtreanalzyer/BytesExprPcodeEmulator.java:124: error: method does not override or implement a method from a supertype
        @Override
        ^
  /Users/gavin/Git/QtREAnalyzer/QtREAnalzyer/src/main/java/qtreanalzyer/BytesExprPcodeEmulator.java:130: error: method does not override or implement a method from a supertype
        @Override
        ^
  3 errors

* Try:
> Check your code and dependencies to fix the compilation error(s)
> Run with --scan to get full insights.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.13/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 5s
1 actionable task: 1 executed
```

--- a/QtREAnalzyer/src/main/java/qtreanalzyer/BytesExprPcodeEmulator.java
+++ b/QtREAnalzyer/src/main/java/qtreanalzyer/BytesExprPcodeEmulator.java
@@ -122,13 +122,13 @@ enum ExprPcodeArithmetic implements PcodeArithmetic<Expr> {
 	}
 
 	@Override
-	public Expr modBeforeStore(int sizeout, int sizeinAddress, Expr inAddress, int sizeinValue,
+	public Expr modBeforeStore(int sizeinOffset, AddressSpace space, Expr inOffset, int sizeinValue,
 			Expr inValue) {
 		return inValue;
 	}
 
 	@Override
-	public Expr modAfterLoad(int sizeout, int sizeinAddress, Expr inAddress, int sizeinValue,
+	public Expr modAfterLoad(int sizeinOffset, AddressSpace space, Expr inOffset, int sizeinValue,
 			Expr inValue) {
 		return inValue;
 	}

